src/enzo/hydro_rk/EOS.h

```diff
@@ -16,6 +16,7 @@ inline void EOS(float &p, float &rho, float &e, float &h, float &cs, float &dpdr
        6: minimum pressure similar (but not equal to)
           http://adsabs.harvard.edu/abs/2004ApJ...606...32R
           equation 4
+       7: Burkert & Bodenheimer Test. Fedderath et al. (2010) Eqn 
```

Comment:
Small typo: should be Federrath et al. 2010, Eqn 30-31. Alternatively, can add the link for clarity https://ui.adsabs.harvard.edu/abs/2010ApJ...713..269F

Reply:
Done